patches/minecraft/net/minecraft/world/World.java.patch

```diff
@@ -168,6 +177,15 @@
                  }
                  catch (Throwable throwable)
                  {
+@@ -543,7 +595,7 @@
+                         {
+                             try
+                             {
+-                                return String.format("ID #%d (%s // %s)", Block.func_149682_b(p_190529_2_), p_190529_2_.func_149739_a(), p_190529_2_.getClass().getCanonicalName());
++                                return String.format("ID #%d (%s // %s // %s)", Block.func_149682_b(p_190529_2_), p_190529_2_.func_149739_a(), p_190529_2_.getClass().getName(), p_190529_2_.getRegistryName());
```

Comment:
This one is fine because if getRegistryName returns null it'll just output "null" and not crash.